src/components/LoginForm.jsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import Form from "./FormBuilder";
+
+const FORM_CONFIG = {
+  onSubmit: () => {
+    console.log("submit");
+    alert("hey");
+  },
+  wrapperClass: "dd",
+  id: "contact-form",
+  title: { text: "Form", class: "ll" },
+  submitBtn: {
+    id: "submit-form-btn-id",
+    content: "Submit",
+    class: "sub"
+  },
+  fields: {
+    firstName: {
+      type: "text",
+      id: "firstName",
+      value: "",
+      label: "First Name",
+      placeholder: "",
+      classes: {
+        wrapperClass: "a",
+        inputClass: "b",
+        errorClass: "c"
+      },
+      validations: [
+        "req",
+        {
+          type: "req",
+          msg: "required field",
+          args: []
+        }
+      ],
+      onBlur: () => {},
+      onFocus: () => {},
+      onChange: () => {
+        console.log("in onChange func");
+      }
+    },
+    lastName: {
+      type: "text",
+      id: "lastName",
+      label: "Last Name",
+      value: "",
+      placeholder: "last name",
+      classes: {
+        wrapperClass: "k",
+        inputClass: "l",
+        errorClass: "k"
+      },
+      validations: [
+        {
+          type: "minLen",
+          args: [3]
+        }
+      ],
+      onBlur: () => {},
+      onFocus: () => {},
+      onChange: () => {
+        console.log("in onChange func");
+      }
+    },
+    email: {
+      type: "email",
+      id: "email",
+      label: "Email",
+      value: "",
+      placeholder: "Enter your email",
+      classes: {
+        wrapperClass: "k",
+        inputClass: "l",
+        errorClass: "k"
+      },
+      validations: [
+        {
+          type: "email",
+          args: []
+        }
+      ],
+      onBlur: () => {},
+      onFocus: () => {},
+      onChange: () => {
+        console.log("in onChange func");
+      }
+    },
+    vehicle: {
+      id: "vehicle",
```

Comment:
Lets just keep id rather than name for simplicity

Reply:
done